x-pack/plugins/security_solution/server/lib/telemetry/endpoint.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ISavedObjectsRepository } from 'src/core/server';
+import { AgentMetadata } from '../../../../ingest_manager/common/types/models/agent';
+import { getFleetSavedObjectsMetadata, getFleetEventsSavedObjects } from './fleet_saved_objects';
+
+export interface AgentOSMetadataTelemetry {
```

Comment:
:red_circle: doc comments on exports

Reply:
Added it for the functions. Not sure if it's needed for the types? But will add if @oatkiller thinks so here